src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -143,6 +143,9 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) :
       root.pendingContext = getContextForSubtree(parentComponent);
       // TODO: Use pending work/state instead of props.
       root.current.pendingProps = element;
+      if (root.current.alternate) {
```

Comment:
This seems wrong to me. I can't really reason about where this will break down. We should probably just move this to the updateQueue instead.